client/client.go

```diff
@@ -187,11 +190,24 @@ func loadServerTrust(conf Config) (*x509.CertPool, error) {
 	// Read certificate file.
 	servcert, err := ioutil.ReadFile(conf.ServerCert)
 	if err != nil {
+		log.Error("/etc/mender/server.crt is inaccessible.")
 		return nil, err
 	}
 
 	if len(servcert) == 0 {
-		return nil, errors.New("unable to find system and server certificates")
+		log.Error("/etc/mender/server.crt is empty.")
```

Comment:
`log.Errorf("Both %s and the system certificate pool are empty.", conf.ServerCert)`

Reply:
Fixed!